### PR TITLE
forms fine tuning and "contrast-prefers: more"

### PR DESCRIPTION
**Pull request by klevain**
A rescue of this [closed PR](https://github.com/transmission/transmission/pull/5333) and adding support for high contrast theme, [as discussed there](https://github.com/transmission/transmission/pull/5333#issuecomment-1494799930) too.

The contrast feature sniffes the user value of prefers-contrast setting then setup the default value in the transmission cookie/prefs. A switch in the burger nav allow the user to switch it off/on.

Here some screenshots :

high-contrast-dark-navhigh-contrast-darkhigh-contrast-lightlow-contrast-dark-navlow-contrast-light

### CODE REVIEW

File: third-party/libdeflate
Comment (ckerr) [inline third-party/libdeflate:1]: This PR shouldn't be changing libdeflate

File: web/public_html/transmission-app.js
Comment (ckerr) [inline web/public_html/transmission-app.js:1]: Please don't change transmission-app in your PRs; a bot generates this file automatically

**Review by ckerr — CHANGES_REQUESTED** [review]
There's so much going on in this PR. It changes the input dialogs, it adds contrast mode, it's fixing breaking spaces, etc.

Would it be possible to break this into single-topic PRs? There's some of it, e.g. the fixes, that I'd like to include in a 4.0.x release but other enhancements that probably don't make sense for a bugfix release.

Marking as "Request changes" but for the record I like the direction this code is going & hope use all these pieces when all is said & done

**Comment by klevain** [comment]
I close this one in favor of [this one](https://github.com/transmission/transmission/pull/5470#event-9130968115).

**Pull request closed without merging**